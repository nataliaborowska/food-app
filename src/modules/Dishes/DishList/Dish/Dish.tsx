import React from 'react';

interface IPropTypes {
  name: string;
  description: string;
  size: number;
  price: number;
}

export const Dish: React.FC<IPropTypes> = (props) => (
  <tr>
    <td>{props.name}</td>
    <td>{props.description}</td>
    <td>{props.size}</td>
    <td>{props.price}</td>
  </tr>
)