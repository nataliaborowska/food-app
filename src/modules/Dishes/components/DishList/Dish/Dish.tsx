import React from 'react';

import {IDish} from '../../../../../store/reducers/dishesReducer';

export const Dish: React.FC<IDish> = (props) => (
  <tr data-test="dish">
    <td>{props.name}</td>
    <td>{props.description}</td>
    <td>{props.size}</td>
    <td>{props.type}</td>
    <td>{props.price}</td>
  </tr>
)