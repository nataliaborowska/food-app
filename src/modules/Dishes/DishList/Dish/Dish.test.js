import {shallow} from 'enzyme';

import {Dish} from './Dish';

describe('testing dish', () => {
  const dishes = {
    name: 'test',
    description: 'test',
    size: 2,
    price: 2,
    _id: 'test',
  };

  it('renders properly', () => {
    const wrapper = shallow(<Dish dishes={dishes} />);

    expect(wrapper).toMatchSnapshot();
  });
});