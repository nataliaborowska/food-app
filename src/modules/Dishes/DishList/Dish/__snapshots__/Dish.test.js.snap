// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<tr>
  <td />
  <td />
  <td />
  <td />
</tr>
`;

exports[`testing dish renders properly 1`] = `
<tr>
  <td />
  <td />
  <td />
  <td />
</tr>
`;
